src/coreservices.h

```diff
@@ -152,6 +152,16 @@ class CoreServices : public QObject {
     std::shared_ptr<mixxx::ScreensaverManager> m_pScreensaverManager;
 
     std::unique_ptr<ControlPushButton> m_pTouchShift;
+    std::unique_ptr<ControlObject> m_pSkinSettingsControl;
```

Comment:
This one should also be a ControlPushButton

Reply:
Please see [@Holzhaus's review above](https://github.com/mixxxdj/mixxx/pull/4647#pullrequestreview-862954338) and maybe shed a light on the ControlObject / ControlPushButton distinction.